components/tools/OmeroWeb/omeroweb/webgateway/static/webgateway/js/ome.roidisplay.js

```diff
@@ -139,10 +139,10 @@ $.fn.roi_display = function(options) {
             var arrowPoint1x, arrowPoint1y, arrowPoint2x, arrowPoint2y;
             // if line starts with arrow...
             if (arrowStart) {
-                arrowPoint1x = x1 - (f * Math.sin(lineAngle - 0.4) * headSize);
-                arrowPoint1y = y1 - (f * Math.cos(lineAngle - 0.4) * headSize);
-                arrowPoint2x = x1 - (f * Math.sin(lineAngle + 0.4) * headSize);
-                arrowPoint2y = y1 - (f * Math.cos(lineAngle + 0.4) * headSize);
+                arrowPoint1x = x1 - (f * Math.sin(lineAngle - 0.35) * headSize);
```

Comment:
couldn't we have a value n = 0.35 and maybe 2 variables v1 = f * Math.sin(lineAngle - n) * headSize and v2 = f * Math.cos(lineAngle - n) * headSize

so if we decide to change we do not have to update 8 lines since one line could be easily missed and introduced a bug